cli/CHANGELOG.md

```diff
@@ -1,8 +1,12 @@
 <!-- See the ../guides/writing-the-cypress-changelog.md for details on writing the changelog. -->
-## 13.13.3
+## 13.14.0
 
 _Released 8/13/2024 (PENDING)_
 
+**Features**
```

Comment:
```suggestion
**Features:**
```

See error message in https://github.com/cypress-io/cypress/actions/runs/10289124114/job/28476115879?pr=29678